packages/pasteup/icons/video-icon.svg

```diff
@@ -1 +1 @@
-<svg width="36" height="23" viewBox="0 0 36 23"><path d="M3.2 0l-3.2 3.3v16.4l3.3 3.3h18.7v-23h-18.8m30.4 1l-8.6 8v5l8.6 8h2.4v-21h-2.4"/></svg>
```

Comment:
Perhaps we should set the width and height in the styles instead to be more explicit

Reply:
@SiAdcock  I just followed the convention from Alex's previous PR. The height is supposedly set on line 101 of OnwardContainer.tsx. I'll look into whether or not it's actually being impacted by that or the svg file